background.js

```diff
@@ -1,36 +1,86 @@
 ﻿/*jshint esversion: 8 */
 
+const MOCK_ASSET_LIST = `[
```

Comment:
Added mock data so that the extension can be checked without requiring the server apis.